reckling tangible landscape activity

skreckli opened the pull request:
Add tangible landscape activity

skreckli committed:
reckling tangible landscape activity

Add tangible landscape activity

wenzeslaus commented on activities/sreckling.py:
```diff
@@ -0,0 +1,60 @@
+import grass.script as gs
+
+# Edit here:
+def runsunshadow(scanned_elev, env, **kwargs):
```
See line 7 above or the _File content / Python files_ check:

```
... File activities/sreckling.py does not contain any functions with the 'run_' prefix
```

skreckli committed:
Update sreckling.py

skreckli committed:
Update sreckling.py

skreckli committed:
Update sreckling.py

petrasovaa reviewed (changes requested):
If you check the rendered result of your PR (see [instructions](https://github.com/ncsu-geoforall-lab/gis714-2022-tangible-landscape#after-opening-pr)), the image is empty because there are no shadows for that landscape at 10.30. So I suggest changing that to e.g. 7am. We can place e.g. a building on the physical model, so the shading will be more visible then, but changing that time will overall result in more interesting results.

Also, it may be useful to compute shaded relief and add it to the rendered layers in the json file.

skreckli committed:
Update sreckling.py

skreckli committed:
Update sreckling.py

skreckli committed:
Update sreckling.py

skreckli committed:
Update sreckling.json

skreckli committed:
Update sreckling.json

skreckli commented:
Thanks for the suggestions Anna.  I've adjusted the time to 8:30am and added a function for r.relief.  I realize that the output of r.sunmask is empty when the input is the elevation.  I was going to bring blocks to cast shadows but I think using the buildings you have sounds even better!

petrasovaa commented on activities/sreckling.json:
```diff
@@ -0,0 +1,21 @@
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=shadow"],
+        ["d.rast", "map=relief"]
```
```suggestion
        ["d.rast", "map=relief"],
        ["d.rast", "map=shadow"]
```

petrasovaa committed:
Reverse layer order
